ani.ter-markosyan/repo/Lesson_3/exercise_17/exercise_17.py

```diff
@@ -0,0 +1,17 @@
+number1, number2, number3 = map(int,input("Input three different numbers: ").split())
```

Comment:
RW: This is good but you should use only the techniques that you learned in Lesson 3.